OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -250,7 +250,7 @@ protected function httpRequest($url, $content = null, array $headers = [], $meth
 
         $headers += array('User-Agent' => 'HWIOAuthBundle (https://github.com/hwi/HWIOAuthBundle)');
         if (is_string($content)) {
-            $headers += array('Content-Length' => strlen($content));
+            $headers += array('Content-Length' => strval(strlen($content)));
```

Comment:
Can you replace this with:
```php
strlen((string) $content))

Reply:
Hi,

Of course, thanks, but it should be: (string)(strlen($content)), isn't it ?

Reply:
Right but in fact it should should be:
```php
(string) strlen($content);